### DISCUSSION TITLE
Consent for each cookie of a category

### DISCUSSION THREAD

**Discussion by DavMoo** [orestbida/cookieconsent#378] [Q&A]
Dear Orest,

is it already possible (somehow) or would it be an idea to make it possible to give consent for each cookies within a category?
I guess it is not a "must have" regarding the GDPR but would be a nice feature to have.

Looking forward to read about your answer.

Thank you for your great work!

David

**Comment by orestbida** [comment]
Hi @DavMoo,

perhaps you meant to disable/enable individual services (rather than individual cookies)? If that's the case,yes you can achieve a more granular control if you upgrade to [v3](https://cookieconsent.orestbida.com/), though you should be aware of the [pros & cons](https://cookieconsent.orestbida.com/essential/introduction.html) before switching.

You can check out a live/production version of cookieconsent v3 on my website:

![sss](https://user-images.githubusercontent.com/34487268/190467099-9ce47c76-4f08-4a10-b27b-96b488d15ad9.PNG)

**Reply by DavMoo** [reply]
Hey @orestbida ,
thats exactly what i wanted. But in your example i would like to see the service/cookie details (name, lifetime etc.).
Basically a combination out of the v2 demo and this (v3) one.

Thanks!

**Reply by orestbida** [reply]
You can always create a cookie table, just like in v2.

![ewewe](https://user-images.githubusercontent.com/34487268/190622747-fa4e32b1-24c9-48ec-b678-90d86197b3d4.PNG)

**Reply by DavMoo** [reply]
Ah ok. I thought the details will be visible next to the service name. But i guess this is also fine.
Thank you. We are planning to implement your cookie consent into our company website.
Will let you know ;-)

Cheers and thanks for support!

**Reply by orestbida** [reply]
I'd generally expect the user to implement either a list of services or a list of cookies (though there's nothing wrong with implementing both). I just don't want to force them to create a service in order to have a cookie table.

You're welcome!

**Comment by yannikbiz** [comment]
Hi thats also what i need but can you show an example ? On how to turn on/off  all single cookies.
I try here but its not working:
https://yannik.biz/test.php

**Reply by orestbida** [reply]
Please open a new discussion so that we don't flood the OP with notifications.
